src/plugins/data/public/query/query_string/query_string_manager.ts

```diff
@@ -63,6 +63,11 @@ export class QueryStringManager {
     return this.storage.get('userQueryString') || '';
   }
 
+  private getDefaultDatasetQueryString(dataset: Dataset, language: string) {
```

Comment:
this appears to get actaully get the initial dataset query string.

`default` within this context is called sparingly compared the get intial query. we should update the name because it will become confusing

Reply:
Updated the name to use `getInitialQueryString`